scripts/all-tests.sh

```diff
@@ -12,7 +12,7 @@ set -e
 
 # Violations thresholds for failing the build
 export PYLINT_THRESHOLD=5650
-export JSHINT_THRESHOLD=9390
+export JSHINT_THRESHOLD=9080
```

Comment:
Is this saying we only have 310 calls to gettext in all our JavaScript? That seems very low...